packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -0,0 +1,330 @@
+// eslint-disable max-lines
```

Comment:
This file is currently a bit long, but we will be iterating and will tidy up as we go.